tiny-build/Dockerfile

```diff
@@ -60,5 +61,8 @@ COPY --from=compose /usr/local/bin/docker-compose /bin/docker-compose
 COPY --from=kubectl /usr/local/bin/kubectl /usr/local/bin/kubectl
 COPY --from=helm /usr/local/bin/helm /usr/local/bin/helm
 COPY --from=cc-test-reporter /bin/cc-test-reporter /bin/cc-test-reporter
+COPY --from=google-cloud-sdk /google-cloud-sdk/bin/ /usr/local/bin/
+COPY --from=google-cloud-sdk /google-cloud-sdk/lib/ /usr/local/lib/
+COPY --from=google-cloud-sdk /google-cloud-sdk/platform/ /usr/local/platform/
```

Comment:
Any reason not to just copy all of `/google-cloud-sdk/*` to `/usr/local/`?

Reply:
That would work, however it will copy a bunch of content that's not required for simply having access to `gcloud` and `gsutil`!